dockstore-webservice/src/main/resources/migrations.1.8.0.xml

```diff
@@ -66,4 +80,10 @@
         <addUniqueConstraint columnNames="imageid" constraintName="uk_imageid" tableName="entry_version_image"/>
         <addForeignKeyConstraint baseColumnNames="imageid" baseTableName="entry_version_image" constraintName="fk_entryVersionImage" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="image"/>
     </changeSet>
+    <changeSet author="esoth (generated)" id="1573241458872-1">
```

Comment:
There should only be one changeset in this entire PR and it should be at the very end. Also, the ID should be human-readable.